Contents/Code/__init__.py

```diff
@@ -122,18 +122,34 @@ def getRecapItems(videos):
 		for video in videos:
 			bitrate = int(video["name"].split("_")[1][0:-1])
 			height = int(video["height"])
-			objects.insert(0, MediaObject(
-				container = Container.MP4,
-				video_codec = VideoCodec.H264,
-				audio_codec = AudioCodec.AAC,
-				video_resolution = height,
-				audio_channels = 2,
-				height = height,
-				width = int(video["width"]),
-				parts = [
-					PartObject(key=Callback(PlayRecap, url=video["url"]))
-				]
-			))
+			if int(Prefs['quality'][0:3]) == height:
```

Comment:
The adapted code does not work with "auto" setting.

```
2017-10-12 23:44:46,655 (70000a906000) :  CRITICAL (runtime:889) - Exception (most recent call last):
  File "/Applications/Plex Media Server.app/Contents/Resources/Plug-ins-1bf240a65/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 843, in handle_request
    result = f(**d)
  File "/Users/pk/Library/Application Support/Plex Media Server/Plug-ins/Lazyman.bundle/Contents/Code/__init__.py", line 303, in Feeds
    oc.add(getRecapVCO(date, "recaps", r, sport))
  File "/Users/pk/Library/Application Support/Plex Media Server/Plug-ins/Lazyman.bundle/Contents/Code/__init__.py", line 166, in getRecapVCO
    items = getRecapItems(recap.videos)
  File "/Users/pk/Library/Application Support/Plex Media Server/Plug-ins/Lazyman.bundle/Contents/Code/__init__.py", line 125, in getRecapItems
    if int(Prefs['quality'][0:3]) == height:
ValueError: invalid literal for int() with base 10: 'Aut'
```

Reply:
Whoops, that was dumb. Fix pushed.